test/e2e/flowaggregator_test.go

```diff
@@ -515,51 +515,41 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 				checkPodAndNodeData(t, record, "perftest-a", controlPlaneNodeName(), "perftest-c", workerNodeName(1))
 				checkFlowType(t, record, ipfixregistry.FlowTypeInterNode)
 			}
-
 			if checkService {
 				if isIntraNode {
 					if !strings.Contains(record, "antrea-test/perftest-b") {
 						t.Errorf("Record with ServiceIP does not have Service name")
 					}
 				} else {
-					checkPodAndNodeData(t, record, "perftest-a", controlPlaneNodeName(), "perftest-c", workerNodeName(1))
-					checkFlowType(t, record, ipfixregistry.FlowTypeInterNode)
-				}
-				assert := assert.New(t)
-				if checkService {
-					if isIntraNode {
-						if !strings.Contains(record, "antrea-test/perftest-b") {
-							t.Errorf("Record with ServiceIP does not have Service name")
-						}
-					} else {
-						if !strings.Contains(record, "antrea-test/perftest-c") {
-							t.Errorf("Record with ServiceIP does not have Service name")
-						}
+					if !strings.Contains(record, "antrea-test/perftest-c") {
```

Comment:
Could we follow the same format as using `assert.Contains` like in L532 and other places?